core/python3AiActionLoop/Dockerfile

```diff
@@ -37,7 +37,7 @@ RUN curl -sL \
 FROM tensorflow/tensorflow:1.15.2-py3-jupyter
 
 # select the builder to use
-ARG GO_PROXY_BUILD_FROM=release
+ARG GO_PROXY_BUILD_FROM=source
```

Comment:
revert

Reply:
Here I need a PR that is not yet avalable as a release